features/admin.identity-verification-providers.v1/pages/identity-verification-provider-edit-page.tsx

```diff
@@ -103,6 +105,9 @@ const IdentityVerificationProviderEditPage: FunctionComponent<IDVPEditPagePropsI
     const hasIdVPDeletePermissions: boolean = useRequiredScopes(
         featureConfig?.identityVerificationProviders?.scopes?.delete);
 
+    const { UIConfig } = useUIConfig();
+    const connectionResourcesUrl: string = UIConfig?.connectionResourcesUrl;
```

Comment:
Here we are getting the base path for connection resources, aren't we? if yes, let's update the variable name to reflect that. (eg: connectionImageBasePath)

Reply:
Since we’re getting the base path and identity verification providers fall under "connections," it aligns with existing conventions to use `connectionResourcesUrl`, as this is the same approach used in authenticators. 

Example : 
 https://github.com/wso2/identity-apps/blob/6f035aefae8e757954093100f8efac5c158e8ad3/features/admin.connections.v1/components/authenticator-grid.tsx#L432

Hence IMO it would be better to stick to `connectionResourcesUrl`.